components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -78,8 +78,11 @@ export function onContextMenuClicked (info: chrome.contextMenus.OnClickData, tab
       break
     case 'elementPickerMode': {
       chrome.tabs.query({ active: true, currentWindow: true }, (tabs: [chrome.tabs.Tab]) => {
-        if (tabs.length > 0) {
-          chrome.tabs.sendMessage(tabs[0].id!, { type: 'elementPickerLaunch' })
+          const tabId = tabs[0]?.id;
+          if (tabId !== undefined) {
+              chrome.scripting.executeScript({target : {tabId},
```

Comment:
doesn't specify world, but default is isolate so that seems fine if the default never changes

Reply:
We plan to migrate to C++ inject soon.